api/http/handler/auth/logout_oauth.go

```diff
@@ -0,0 +1,85 @@
+package auth
+
+import (
+	"fmt"
+	. "github.com/portainer/libhttp/error"
```

Comment:
dot imports are useful if used sparsely and with purpose. I fail to see the reason for this in this file. In my book, shortcutting yet another`error` package does not count into this. Also, you already aliased the package `github.com/portainer/portainer/api` to `portainer`, and so you should do with the error package.

I think something along the lines of `portError` should suffice.